CodeGeneration/Sources/SyntaxSupport/Traits.swift

```diff
@@ -56,7 +56,13 @@ public let TRAITS: [Trait] = [
       Child(name: "asyncSpecifier", kind: .token(choices: [.keyword(.async), .keyword(.reasync)]), isOptional: true),
       Child(name: "unexpectedBetweenAsyncSpecifierAndThrowsSpecifier", kind: .node(kind: .unexpectedNodes), isOptional: true),
       Child(name: "throwsSpecifier", kind: .token(choices: [.keyword(.throws), .keyword(.rethrows)]), isOptional: true),
-      Child(name: "unexpectedAfterThrowsSpecifier", kind: .node(kind: .unexpectedNodes), isOptional: true),
+      Child(name: "unexpectedBetweenThrowsSpecifierAndThrownType", kind: .node(kind: .unexpectedNodes), isOptional: true),
```

Comment:
This is an API-breaking change that we should mention in the [API-Incompatible Changes](https://github.com/apple/swift-syntax/pull/2219/files#api-incompatible-changes) section of the release notes. I think it’s fine to break the API here since it’s unlikely that this property is in considerable use.